processor/src/fast/mod.rs

```diff
@@ -965,6 +968,42 @@ impl FastProcessor {
         Ok(())
     }
 
+    /// Analogous to [`Process::resolve_external_node`](crate::Process::resolve_external_node), but
+    /// for asynchronous execution.
+    async fn resolve_external_node_async(
```

Comment:
```suggestion
    async fn resolve_external_node(
```
(I hate to do this, but we don't use `_async` pre/post-fixes in the fast processor)